Tasks/ANTV1/Tests/L0FailWithValidInputs.ts

```diff
@@ -0,0 +1,15 @@
+import path = require('path');
+import { TaskMockRunner }  from 'azure-pipelines-task-lib/mock-run';
+import * as answers from './answers'
```

Comment:
nit: 
extra space here  `}  from` in some files and probably we need to ned line with `;` :)

Reply:
Fixed it. Thanks!